### PR TITLE
Update readme to mention FFmpeg 2.8.11.

### PR DESCRIPTION
**Pull request by rosiel**
**JIRA Ticket**: none

# What does this Pull Request do?

Mentions that it works with FFmpeg 2, as well as fixes what appears to be "our suggestion" of using libfaac to state that the default is there purely for legacy reasons. 

# What's new?

couple paras in the readme

# How should this be tested?

Get someone to install this on Bionic (Ubuntu 18.04) based on the README. When they discover that 1.1.4 won't compile on that distro, and libfaac and other plugins have been deprecated in later versions of ffmpeg, they should NOT conclude (as was concluded prior to this pull request) that Video Solution Pack is not compatible with Bionic. 

# Additional Notes:

Can we please change how the Vagrant works, as that is also _in practice_ a default set up of islandora, to not install 1.1.4? 

# Interested parties
@Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by rosiel** [comment]
Restarted Travis post #152 (thanks for all your work on sorting out codesniffer @whikloj @jonathangreen @DiegoPino @adam-vessey @dannylamb !!) 

Just waiting for that green light... 
Lorde

**Comment by whikloj** [comment]
@rosiel this will probably fail again until #152 is merged. But then you'll be golden.

**Comment by rosiel** [comment]
oh hey, you're right, it's not merged yet. Odd... the travis is turning green except on two oddballs (with different fedora and php versions).

**Comment by whikloj** [comment]
Yeah, the code here is pretty clean but also things like [this](https://travis-ci.org/Islandora/islandora_solution_pack_video/jobs/489630992#L1310-L1319) are happening so its a bit of a false positive.

That being said this is a README PR and seems good to me.

**Review by whikloj — APPROVED** [review]
👍 24 🕐 started

**Comment by rosiel** [comment]
Thanks @whikloj . I'm gonna be so embarrassed if there's whitespace errors. 😶

**Comment by whikloj** [comment]
`*.md` and `*.txt` are not scanned, so you're clear 😄

**Pull request merged**
